bundles/org.openhab.binding.daikin/src/main/java/org/openhab/binding/daikin/internal/api/Enums.java

```diff
@@ -138,4 +137,75 @@ public String getValue() {
             return value;
         }
     }
+
```

Comment:
Please add yourself to the author tags for this file.

Reply:
Fixed.